### PR TITLE
add vlan_expander filter

### PR DESCRIPTION
**Pull request by akira6592**
##### SUMMARY

add `vlan_expander` filter that  inverse of `vlan_parser`.

example:

```yaml
    - name: vlan_expander
      debug:
        msg: "{{ '1,10-12,15,20-22' | ansible.netcommon.vlan_expander }}"

        # "msg": [
        #     1,
        #     10,
        #     11,
        #     12,
        #     15,
        #     20,
        #     21,
        #     22
        # ]
```
My use case is to find out if the allowed vlan gotten from the device has a specific vlan.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- `ansible.netcommon.vlan_expander`

### CODE REVIEW

File: changelogs/fragments/280-vlan_expander.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
```
Comment (Qalthos) [inline changelogs/fragments/280-vlan_expander.yaml:2]:
```suggestion
minor_changes:
```
Major changes are backwards incompatible changes to existing features, which this is not

**Review by Qalthos — CHANGES_REQUESTED** [review]

File: changelogs/fragments/280-vlan_expander.yaml (hunk 1 above)
Comment (akira6592) [reply changelogs/fragments/280-vlan_expander.yaml]: Thank you.  I committed it.

**Review by GomathiselviS — CHANGES_REQUESTED** [review]
The PR looks good to me. Please fix the linter failures.

**Comment by akira6592** [comment]
@GomathiselviS Thank you. I fixed it.

**Review by Qalthos — APPROVED** [review]

**Review by GomathiselviS — APPROVED** [review]

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
Due to change in our pipeline configuration for zuul, we'll be closing this PR in favor of https://github.com/ansible-collections/ansible.netcommon/pull/299

This is because we have no easy way to remove the old status api checks.

**Pull request closed without merging**